src/commands/e2e/ConfigTemplate.ts

```diff
@@ -73,7 +73,7 @@ export class ConfigTemplate {
                 fs.mkdirSync(screenshotDir, { recursive: true });
             }
 
-            const filePath = path.join(
+            var filePath = path.join(
```

Comment:
[Question]
Why replace the const with var?

Reply:
cplace project is still on an older javascript version and highlights const and let declarations so I switched it back